pyannote/audio/interactive/diff/recipe.py

```diff
@@ -0,0 +1,239 @@
+            files_errors[filename], excerpt, Segment(0, duration)
+        )
+        ref = get_audio_spans(reference[filename], excerpt, Segment(0, duration))
+        hyp = get_audio_spans(hypothesis[filename], excerpt, Segment(0, duration))
+
+        yield {
+            "path": path,
+            "text": text,
+            "audio": audio,
+            "audio_spans": audio_spans,
+            "reference": ref,
+            "hypothesis": hyp,
+            "chunk": {"start": excerpt.start, "end": excerpt.end},
+            "meta": {
+                "file": filename,
+                "start": f"{excerpt.start:.1f}",
+                "end": f"{excerpt.end:.1f}",
+            },
+        }
+
+
+@prodigy.recipe(
+    "pyannote.diff",
+    dataset=("Dataset to save annotations to", "positional", None, str),
+    source=(
+        "Path to directory containing audio files whose annotation is to be checked",
+        "positional",
+        None,
+        str,
+    ),
+    reference=("Path to reference file", "positional", None, str),
+    hypothesis=("Path to hypothesis file ", "positional", None, str),
+    chunk=(
+        "Split long audio files into shorter chunks of that many seconds each",
+        "option",
+        None,
+        float,
+    ),
+    minduration=("Minimum duration of errors in ms", "option", None, int),
+    diarization=(
+        "Optimal one-to-one mapping between reference and hypothesis",
+        "flag",
+        None,
+        bool,
+    ),
+    falsealarm=("Display false alarm errors", "flag", None, bool),
+    confusion=("Display confusion errors", "flag", None, bool),
+    misseddetection=("Display missed detection errors", "flag", None, bool),
+)
+def diff(
+    dataset: str,
+    source: Union[str, Iterable[dict]],
+    reference: str,
+    hypothesis: str,
+    chunk: float = 30.0,
+    minduration=200,
+    diarization: bool = False,
+    falsealarm: bool = False,
+    confusion: bool = False,
+    misseddetection: bool = False,
+) -> Dict[str, Any]:
+
+    recipe_dir = Path(__file__).resolve().parent
+    common_dir = recipe_dir.parent / "common"
+
+    controllerDiff = recipe_dir / "controller.js"
+    legend = recipe_dir / "legend.html"
+    wavesurfer = common_dir / "wavesurfer.js"
+    regions = common_dir / "regions.js"
+    html = common_dir / "template.html"
+    css = common_dir / "template.css"
+
+    with open(controllerDiff) as sc_diff, open(wavesurfer) as s_wavesurfer, open(
+        regions
+    ) as s_regions, open(html) as f_html, open(legend) as f_legend, open(css) as f_css:
+        script_text = s_wavesurfer.read()
+        script_text += "\n" + s_regions.read()
+        script_text += "\n" + sc_diff.read()
+        templateH = f_html.read()
+        templateL = f_legend.read()
+        templateC = f_css.read()
+
+    ref = util.load_rttm(reference)
+    hyp = util.load_rttm(hypothesis)
+    listerrors = [falsealarm, confusion, misseddetection]
+
+    hstream = (
```

Comment:
`hstream` stands for what? `hashed_stream`? 
I'd rather use the latter for readability.